Emby.Server.Implementations/HttpServer/ResponseFilter.cs

```diff
@@ -41,11 +41,11 @@ public void FilterResponse(IRequest req, HttpResponse res, object dto)
                 res.Headers.Add(key, value);
             }
             // Try to prevent compatibility view
-            res.Headers["Access-Control-Allow-Headers"] = ("Accept, Accept-Language, Authorization, Cache-Control, " +
+            res.Headers["Access-Control-Allow-Headers"] = "Accept, Accept-Language, Authorization, Cache-Control, " +
```

Comment:
I don't believe this is an issue and it was classed as an uneccessay parenthesis, but it looks like it could cause troubles 🤷